generate_span_wagner_tables.py

```diff
@@ -0,0 +1,24 @@
+"""
+This script is used to generate the tables included in open_petro_elastic.material.span_wagner.tables. It serves as
+documentation for how the tables were generated, as well as a template for generating tables of different resolution or
+bounds.
+"""
+import os
+import numpy as np
+from open_petro_elastic.material.span_wagner.tables.lookup_table import generate_lookup_table
+from open_petro_elastic.material.span_wagner.carbon_dioxide import carbon_dioxide_density
+from time import perf_counter
+
+
+t_density = 273.15 + np.hstack((np.arange(0, 50, step=0.05), np.arange(50, 200.1, step=0.5)))
+p_density = np.hstack((np.arange(0.5, 10, step=0.01), np.arange(10, 100.1, step=0.1)))
+
+out_dir = os.path.join('src', 'open_petro_elastic', 'material', 'span_wagner', 'tables')
+
+t0 = perf_counter()
```

Comment:
It would be good to have a bit of a refactor of this to be able to integrate this into other tooling later:
```python
import argparse
import numpy as np
from open_petro_elastic.material.span_wagner.tables.lookup_table import generate_lookup_table
from open_petro_elastic.material.span_wagner.carbon_dioxide import carbon_dioxide_density


t_density = 273.15 + np.hstack((np.arange(0, 50, step=0.05), np.arange(50, 200.1, step=0.5)))
p_density = np.hstack((np.arange(0.5, 10, step=0.01), np.arange(10, 100.1, step=0.1)))


def run(args):
    generate_lookup_table(
      lambda _t, _p: carbon_dioxide_density(_t, _p, interpolate=False, force_vapor='auto'),
      t_density, p_density, os.path.join(out_dir, 'carbon_dioxide_density.npz')
    return 0


TOOL_DESCRIPTION = """
used to generate the tables included in open_petro_elastic.material.span_wagner.tables.
"""

def make_parser(prog="generate_sp_tables"):
    ap = argparse.ArgumentParser(prog=prog, description=TOOL_DESCRIPTION)
    ap.add_argument(
        "output_file", type=argparse.FileType("r"), help="The output file to write the density to"
    )
   
    return ap
    
    
def parse_arguments(argv):
    ap = make_parser(argv[0])
    return ap.parse_args(argv[1:])


def main():
    sys.exit(run(parse_arguments(sys.argv)))


if __name__ == "__main__":
    main()

```

Reply:
Implemented pretty much as suggested with minor modifications.